airflow/docker.go

```diff
@@ -400,12 +403,21 @@ func (d *DockerCompose) PS() error {
 
 // Kill stops a local airflow development cluster
 func (d *DockerCompose) Kill() error {
+	// Killing an already killed project produces an unsightly warning,
+	// so we briefly switch to a higher level before running the kill command.
+	// We then swap back to the original level.
+	originalLevel := logrus.GetLevel()
+	logrus.SetLevel(logrus.ErrorLevel)
+
 	// Shut down our project
 	err := d.composeService.Down(context.Background(), d.projectName, api.DownOptions{Volumes: true, RemoveOrphans: true})
 	if err != nil {
 		return errors.Wrap(err, composeStopErrMsg)
 	}
 
+	// Restore the log level.
+	logrus.SetLevel(originalLevel)
```

Comment:
Defer this logic right after updating the log level to play it safely.

Reply:
Good call, done!